mypy/stubgen.py

```diff
@@ -723,7 +773,14 @@ def process_member_expr_decorator(self, expr: MemberExpr, context: Decorator) ->
                 self.add_coroutine_decorator(context.func,
                                              expr.expr.name + '.coroutine',
                                              expr.expr.name)
-        return is_abstract
+        elif (isinstance(expr.expr, NameExpr) and
+              (expr.expr.name == 'typing' or
+               self.import_tracker.reverse_alias.get('typing')) and
```

Comment:
Should this be `expr.expr.name in ('typing', self.import_tracker.reverse_alias.get('typing'))`? Maybe add test case that uses `import typing as t` or similar.